### ISSUE TITLE
S3 + S4 - Find us on Social media needs to announce type of social media (for the icon)

### ISSUE THREAD

**Issue by mmafe** [brharp/hjckrrh#114]
Each social media account should be announced via assistive technology as follows:
"Type of account - Name of account"

(We can see the type of social media account, but currently we only hear the name of the account.)

**Comment by brharp** [comment]
Added social media network name for screen readers.
